test/TestAsset/Alpha.php

```diff
@@ -4,21 +4,27 @@
 
 namespace LaminasTest\Filter\TestAsset;
 
-use Laminas\Filter\AbstractFilter;
+use Laminas\Filter\FilterInterface;
 
-use function is_array;
-use function is_scalar;
+use function is_string;
 use function preg_replace;
 
-/** @template-extends AbstractFilter<array{}> */
-class Alpha extends AbstractFilter
+/** @implements FilterInterface<string> */
```

Comment:
I guess the type is here so that we tell what is being return from `filter` function?
If I see correct, `filter´ returns `mixed` here as there is `! is_string` check which returns the value without touching it?